package.json

```diff
@@ -39,7 +39,8 @@
       {
         "command": "onevscode.configuration-settings",
         "title": "configuration-settings",
-        "category": "ONE"
+        "category": "ONE",
```

Comment:
![image](https://user-images.githubusercontent.com/17171963/151299433-f3cc4333-627d-4c84-a3e7-32be72eb3700.png)
By hovering upon the icon, the title text "configuration-settings" appears 
By clicking, one configuration window is opened.

Reply:
TODO: import the cfg file at once